modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/wal/filehandle/FileWriteHandleImpl.java

```diff
@@ -499,7 +493,7 @@ private static void fsync(MappedByteBuffer buf, int off, int len) throws IgniteC
                     if (segs != null) {
                         assert segs.size() == 1;
 
-                        segs.get(0).release();
+                        fsyncReadSegment(segs.get(0));
```

Comment:
New `fsync` invocation here. Do we really need it?

Reply:
Add flag to skip it in this place. Open a ticket to figure out the reason.